Makefile

```diff
@@ -65,6 +65,7 @@ guess_platform_script := \
 guess_platform := $(shell $(guess_platform_script))
 
 all: all-$(guess_platform)
+check: check-$(guess_platform)
 
```

Comment:
At the moment, those two testsuites do their own (evil) platform detection -- but I don't think it's fair to expect you to fix that in this PR. ;-)